project/spreadsheet.py

```diff
@@ -86,22 +265,47 @@ def initialize():
             creds = tools.run_flow(flow, store, flags)
 
         except clientsecrets.InvalidClientSecretsError:
-            print ('The client secrets were missing or invalid: ')
-            cont = False
+            print('The client secrets were missing or invalid: ')
         except client.UnknownClientSecretsFlowError:
             print('This OAuth 2.0 flow is unsupported')
-            cont = False
         except client.Error:
             print('Unexpected Error')
-            cont = False
 
-    if cont:
+    return creds
```

Comment:
We'd need to initialize creds at the top of the function unless tools.run_flow returns some determined value?

Reply:
It is initialized at the top of the function:    creds = store.get()